src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs

```diff
@@ -4208,6 +4208,112 @@ private static string GetCommentExpectedString(bool prettyPrint)
             return Encoding.UTF8.GetString(ms.ToArray());
         }
 
+        [Fact]
+        public void WriteCommentsInArrayAsIndentedJson()
+        {
+            const string expectedIndentedJson =
```

Comment:
Thanks for adding string literal tests. For completeness, would it be possible to add an equivalent one with indentation disabled?

Reply:
Added.